webapp/pom.xml

```diff
@@ -94,7 +94,7 @@
          <plugin>
             <groupId>org.mortbay.jetty</groupId>
             <artifactId>maven-jetty-plugin</artifactId>
-            <version>6.1.10</version>
+            <version>6.1.26</version>
```

Comment:
Yeah, I didn't want to be too disruptive but the latest version of jetty worked for me to. I agree with @djazayeri another issue would be the best place to do this.